client/client_test.go

```diff
@@ -48,7 +49,6 @@ var (
 var (
 	hostIP                             = env("HOST_IP", dockerHostGateway)
 	testCloudImage                     = env("TEST_CLOUD_IMAGE", "ghcr.io/calyptia/cloud")
```

Comment:
```suggestion
	testCloudImage                     = env("TEST_CLOUD_IMAGE", "ghcr.io/calyptia/cloud:main")
```
To serve as example.